src/main/java/uk/co/bconline/ndelius/transformer/SearchResultTransformer.java

```diff
@@ -69,6 +71,18 @@ public SearchResult map(SearchResultEntity entity) {
 				.build();
 	}
 
+	public SearchResult map(UserEntity entity) {
+		return SearchResult.builder()
+				.username(entity.getUsername())
+				.forenames(combineNames(entity.getForename(), entity.getForename2()))
+				.surname(entity.getSurname())
+				.endDate(entity.getEndDate())
+				.staffCode(entity.getStaff() != null ? entity.getStaff().getCode() : null)
+				.teams(entity.getStaff() != null ? teamTransformer.map(entity.getStaff().getTeams()) : null)
```

Comment:
Would it be better to use optionals here? Happy to stick with the original if you think it's more readable.
```suggestion
				.staffCode(ofNullable(entity.getStaff()).map(StaffEntity::getCode).orElse(null))
				.teams(ofNullable(entity.getStaff()).map(StaffEntity::getTeams).map(teamTransformer::map).orElse(null))
```